### ISSUE TITLE
Firefox: Keyboard shortcut fails first time pressed in each tab

### ISSUE THREAD

**Issue by jabranham** [sprig/org-capture-extension#26]
Currently the keyboard shortcut does not work unless the button is physically pressed first. I've got no idea why; related to #19. 

I spent some time trying to get it to work without much luck. It's possibly a bug in Firefox, as it seems to work in Chromium with no problems.

**Comment by sprig** [comment]
I don't understand. Are you saying that after you physically press the
button once, the binding begins to work as well?
-- 


Konstantin Kliakhandler

**Comment by jabranham** [comment]
Yes, that's exactly what I'm saying. Since it works in Chrome(ium) I'm betting this is a Firefox bug. I reported it to Mozilla a few hours ago, here's the report: https://bugzilla.mozilla.org/show_bug.cgi?id=1365176

I'll let you know if/when they follow up.

**Comment by sprig** [comment]
What version of Firefox?
Once it begins working, does it work on all tabs or just the tab where you
initially pressed it?
Does it continue working after you do a hard refresh?
Does it continue working once you close said tab? Window?

-- 
-- 
Konstantin Kliakhandler
    http://slumpy.org
          )°) )°( (°(

**Comment by jabranham** [comment]
1 - 53
2 - No, just the tab where I initially pressed it
3 - No
4 - No

**Comment by sprig** [comment]
Thanks!

It is also possible that I am not initializing the binding/extension
properly. I relied on user reports (and my own short test) to make sure the
extension indeed works on FF as well, but haven't looked at the docs more
more than a glance.

If you wish, you can make sure that there isn't any missing ingredient in
the mix. I will do it myself eventually but it will take me some time (few
weeks probably) as I am overwhelmed with other duties at the moment.
-- 
-- 
Konstantin Kliakhandler
    http://slumpy.org
          )°) )°( (°(

**Comment by sprig** [comment]
@jabranham did you get any response regarding the firefox issue?

**Comment by jabranham** [comment]
They've marked it as a duplicate. I haven't had time yet to make sure it
actually *is* a duplicate, though. The info should be in the link I posted
earlier if you want to look at it.

**Comment by sprig** [comment]
oh, right - I forgot that you had the link there - thanks!

**Comment by PrestanceDesign** [comment]
Same here with Firefox Quantum...nothing happens with keyboard shortcut Ctrl+Shift+L.

**Comment by Fuco1** [comment]
Did they remove the shortcuts in newer firefox? I can't seem to be able to find where to set it anymore.

@sprig If there is some standard way to do this it would be much appreciated to put it in the README, I couldn't find it there either.

**Comment by failable** [comment]
Any new of this?

**Comment by prestancedesign** [comment]
With new Firefox version, you can now customize keyboard shortcuts: https://support.mozilla.org/en-US/kb/manage-extension-shortcuts-firefox

It is what I do, and it works fine now :+1: 
![image](https://user-images.githubusercontent.com/1840522/72142700-899af900-3395-11ea-96b7-43ebe331004a.png)

**Comment by failable** [comment]
@PrestanceDesign Thanks!

**Comment by pedro-nonfree** [comment]
> With new Firefox version, you can now customize keyboard shortcuts: https://support.mozilla.org/en-US/kb/manage-extension-shortcuts-firefox
> 
> It is what I do, and it works fine now 👍 ![image](https://user-images.githubusercontent.com/1840522/72142700-899af900-3395-11ea-96b7-43ebe331004a.png)

@prestancedesign very nice shortcut, regrettably does not work with text selection (which remembers me to the same it happens with thunderlink in thunderbird)

EDIT: ah oh, depends on the website, with element.io and telegram (a and k version) does not work, with whatsapp web works. In general, works with any website, but certain websites do not like this plugin :shrug: 

EDIT 2: I tried other commands with Alt such as Alt+1, Alt+x, etc. and the selection is lost on that applications. **I found another shortcut that preserves the selection and solved the problem: Ctrl+Alt+C**
